File: Python/beginner/1929.py
# https://judge.beecrowd.com/pt/problems/view/1929
# Triângulo

# Main

a, b, c, d = map(int, input().split())

if a + b > c and a + c > b and b + c > a or \
   a + b > d and a + d > b and b + d > a or \
   a + c > d and a + d > c and c + d > a or \
   b + c > d and b + d > c and c + d > b:
    print("S")
else:
    print("N")
    
# By: Fernando Serra
# https://github.com/fernandosserra

File: C/beginner/1929.c
// https://judge.beecrowd.com/pt/problems/view/1929
// Triângulo

// Libraries
#include <stdio.h>

// Function Prototypes
int isTriangle(int x, int y, int z);


// Main
int main (){
    int a, b, c, d;

    scanf("%d %d %d %d", &a, &b, &c, &d);

    if (isTriangle(a, b, c) || 
        isTriangle(a, b, d) || 
        isTriangle(a, c, d) || 
        isTriangle(b, c, d)){
            printf("S\n");
        } else {
            printf("N\n");
        }

    return 0;
}

// Function Bodies
int isTriangle(int x, int y, int z){
    return (x + y > z && x + z > y && y + z > x);
}


// By: Fernando Serra
// https://github.com/fernandosserra